src/Events/Appointments/AppoinmentSetEvent.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ *
+ * @package      OpenEMR
+ * @link               https://www.open-emr.org
+ *
+ * @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @copyright Copyright (c) 2021 Sherwin Gaddis <sherwingaddis@gmail.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ */
+
+namespace OpenEMR\Events\Appointments;
+
+use Symfony\Component\EventDispatcher\Event;
+
+/**
+ * Event object for know what type of appointment has been set on the calendar
+ *
+ * @package OpenEMR\Events
+ * @subpackage Appointments
+ *
+ */
+class AppoinmentSetEvent extends Event
+{
+    /**
+     * This event is triggered after a new patient appointment has been scheduled
+     */
+    const EVENT_HANDLE = 'appointment.set';
+
+    /**
+     * @var
+     */
+    private $post;
+
+    public function __construct($post)
+    {
+        $this->pid = $post;
```

Comment:
is this supposed to be (i am not sure, but just seemed odd):
```
$this->post = $post;
```

Reply:
Yes, I caught that and updated the dev. I will port it over on my next push to the repo. I don't know what other codebase items that will have to be changed. They have not exposed me to the entire project. They are giving me pieces to build. I haven't seen the main code.